format_answers.py

```diff
@@ -45,6 +49,5 @@
     
     # Print formatted results
     print(f"- user response: {user_answer}")
-    print(f"- expected response: {expected_answer}")
```

Comment:
If we don't include the correct answer here, how can we best instruct Cosmo what the correct answer is?  Do we presume the AI models just know?  Or is there a better way?